src/FSharp.UMX.fsproj

```diff
@@ -28,6 +28,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
-    <PackageReference Include="FSharp.Core" Version="4.3.4" />
+    <PackageReference Condition=" '$(TargetFramework)' == 'netstandard2.0' " Include="FSharp.Core" Version="4.3.4" />
+    <PackageReference Condition=" '$(TargetFramework)' == 'net6.0' " Include="FSharp.Core" Version="6.0.7" />
```

Comment:
Need to target a newer version of `FSharp.Core` to pick up the `uint` alias. Again, I chose to do this when compiling for `net6.0` as it felt like it was then safe to assume we could use this later version. If that's not acceptable or if there's a better condition to target than just `net6.0` let me know.